libvmi/accessors.c

```diff
@@ -266,11 +317,18 @@ vmi_get_vmid(
 }
 
 /* convert a kernel symbol into an address */
-status_t vmi_translate_ksym2v (vmi_instance_t vmi, const char *symbol, addr_t *vaddr)
+status_t
+vmi_translate_ksym2v(
+    vmi_instance_t vmi,
+    const char *symbol,
+    addr_t *vaddr)
 {
     status_t status = VMI_FAILURE;
     addr_t address = 0;
 
+    if (!vmi)
```

Comment:
Should check that `symbol` and `vaddr` are not NULL too

Reply:
Ok I will change that. Since They are not pointers I thought we didn't have to check them

Reply:
My mistake, I didn't noticed they were pointers..